inc/import.php

```diff
@@ -199,6 +199,41 @@ function ozh_ta_linkify_tweet( $tweet ) {
             $display_url  = $media->display_url;
             $expanded_url = $media->expanded_url;
             $tco_url      = $media->url;
+
+	    if( $ozh_ta['download_images'] == 'yes' ) {
+                $url = $media_url;
+    		$tmp = download_url( $url );
+    		
+		if( is_wp_error( $tmp ) ){
+           	    // download failed, handle error
```

Comment:
This block does nothing. Instead, it should have an else block enclosing the rest of the code (up to line 235). If we haven't downloaded it there's no point attempting to store into `$file_array` then taking it out again.

Is there a way of reporting failures to the user, maybe by incrementing a counter visible in the theme or on the settings page?